tests/screenshots/cases/software/basic/text_input.slint

```diff
@@ -0,0 +1,29 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+TestCase := Window {
+    width: 64px;
+    height: 64px;
+    forward-focus: i-text-input;
+    VerticalLayout {
+        i-text-input := TextInput {
+            text: "Hello Everyone";
+            wrap: word-wrap;
+            color: green;
+            font-size: 10px;
```

Comment:
This places the cursor between the "E" and the "v" of "Everyone". That might give us better coverage than 0 :)

```suggestion
            font-size: 10px;
            cursor-position-byte-offset: 7;
            anchor-position-byte-offset: 7;
```

Reply:
Yes right ;-), thank you.